src/k8s/pkg/k8sd/types/cluster_config_validate_test.go

```diff
@@ -54,3 +55,49 @@ func TestValidateCIDR(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateLoadbalancer(t *testing.T) {
+	for _, tc := range []struct {
+		cidr      []string
+		expectErr bool
+	}{
+		{cidr: []string{"10.3.0.0/16"}},
+		{cidr: []string{"2001:0db8::/32"}},
+		{cidr: []string{"10.3.0.0/16", "2001:0db8::/32"}},
+		{cidr: []string{"10.3.0.10-10.3.0.20"}},
+		{cidr: []string{"10.3.0.32/28", "10.3.0.10-10.3.0.20"}},
+		{cidr: []string{"2001:0db8::0-2001:0db8::10"}},
+		{cidr: []string{"10.3.0.32/28", "2001:0db8::0-2001:0db8::10"}},
+		{cidr: []string{""}, expectErr: true},
+		{cidr: []string{"bananas"}, expectErr: true},
+		{cidr: []string{"fd01::/64,fd02::/64,fd03::/64"}, expectErr: true},
+		{cidr: []string{"10.3.0.10-10.3.0.300"}, expectErr: true},
+		{cidr: []string{"10.3.0.10-10.3.0.7"}, expectErr: true},
+		{cidr: []string{"2001:0db8::0-2001:0db8::gg"}, expectErr: true},
+		{cidr: []string{"", "10.3.0.10-10.3.0.300"}, expectErr: true},
+		{cidr: []string{"10.3.0.10-10.3.0.12-10.3.0.15"}, expectErr: true},
```

Comment:
```suggestion
		{cidr: []string{"10.3.0.10-10.3.0.12-10.3.0.15"}, expectErr: true},
		{cidr: []string{"10.3.0.15-10.3.0.10"}, expectErr: true},
```

Reply:
Already check has been added in L#75 to verify if start > stop.

Reply:
awesome.  i just didn't spot it.  thank you